Packs/Incapsula/Integrations/Incapsula/Incapsula.yml

```diff
@@ -1545,25 +1310,72 @@ script:
       description: Numeric identifier of the site to operate on
     - name: protected_app
       description: Protect admin areas of joomla | wordpress | phpBB
-    description: Use this operation to configure Login Protect on wordpress | joomla
-      | phpbb admin areas
+    description: Use this operation to configure Login Protect on wordpress | joomla | phpbb admin areas
   # Integration
   - name: incap-get-ip-ranges
     arguments: []
     description: Use this operation to get the updated list of Incapsula IP ranges
   - name: incap-get-texts
     arguments: []
-    description: Use this operation to retrieve a list of all text messages that may
-      be part of API responses
+    description: Use this operation to retrieve a list of all text messages that may be part of API responses
   - name: incap-get-geo-info
     arguments: []
-    description: Use this operation to retrieve a list of all the countries and continents
-      codes
+    description: Use this operation to retrieve a list of all the countries and continents codes
   - name: incap-get-app-info
     arguments: []
     description: Use this operation to retrieve a list of all the client applications
   # Infrastructure Protection Test Alerts
-system: true
+  - arguments:
+    - description: The customer's IP range.
+      name: ip_range
+      required: true
+    - auto: PREDEFINED
+      description: "One of the following: BGP, NETFLOW, PROTECTED_IP\t"
```

Comment:
Please try to explain what each argument does, also in other arguments (the options are already mentioned in the predefined)

Reply:
I do not know and[ Imperva's docs ](https://docs.imperva.com/bundle/cloud-application-security/page/cloud-v1-api-definition.htm) (Traffic Statistics and Logs -> /api/v1/infra/top-table) don't give any more details either